Backend/tests/test_user_service.py

```diff
@@ -100,6 +102,12 @@ def test_generate_new_uid_with_a_previous_uid(
     assert user_service.generate_new_uid(user_1.email) == {"deviceUID": 1}
 
 
+def test_browse_by_page(bu_service):
+    result = bu_service.get_all_users(4, 0)
+
+    assert len(result) == 4
```

Comment:
If there are no users added then len will always be 0 here. Consider the "setup" step for unit tests. Each test is isolated and starts with an empty database. You need to add the information that you consider relevant for this test before trying to assert that there's anything in the database.